### ISSUE TITLE
Artists: Inherit artist main picture from base voicebank

### ISSUE THREAD

**Issue by riipah** [VocaDB/vocadb#1366]
Most of the time this is the same for variations/appends, so there's no need to upload separate pictures for them.

Main issue is how to handle artist picture in listings, as this could be a performance problem. Maybe the value would need to be cached in the DB.

**Comment by CatgirlFrostmoon** [comment]
Mhm, I actually don't think this is a good idea. Someone was recently confused over [IA Talk](https://vocadb.net/Ar/86874) having Aka Akasaka as the illustrator - IA Talk was not directly labelled as such, but instead inherited it. Similarly, [Teto's oldest banks](https://vocadb.net/Ar/176156) automatically inherited TWINDRILL as their manager, despite their life and death pre-dating the group by a good few years. I can not de-set this at all.

In other words: I genuinely think we need to redo the inheritance system (due to misinformation potential) before we add anything more to it,

**Comment by andreoda** [comment]
A solution to one of the issues above could be to implement a "remove image" option or a toggle to active/deactivate the inheritance.

**Comment by CatgirlFrostmoon** [comment]
(Not an image, but the name and link of the illustrator / manager. But yeah, a remove option would help fix this problem.)
